### DISCUSSION TITLE
Colors of forecast low annotation

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3565] [Ideas]
Hello @Der-Schubi 

I wonder what you think about this open issue:  https://github.com/NightscoutFoundation/xDrip/issues/682  
  
I wonder if you may be interested and have the time to put something together.  
I hope we can add 2 items to the xDrip legend (xDrip color settings).  One as the background and one as the foreground of the forecast low alert.  
There are two other colors that are used when there is a forecast that is beyond the trigger point of the alert.  
But, I prefer to minimize the number of items we add.  When the alert goes off, we always have red over black.  The green and yellow over black show when there is no alert and I prefer to leave those two as they are.  

Please tell me what you think. 
Of course you may be busy with life and it is perfectly understandable if you cannot.  
Thanks.

**Comment by Der-Schubi** [comment]
Hi!

I really don't know what would make sense here. We already talked about this [here](https://github.com/NightscoutFoundation/xDrip/pull/2214#pullrequestreview-1033627277). Being able to change the color of the alert with red text would be a good start, as it is hardest to read.
I'll have a look and may put together a PR.
Is there a way to manually trigger those messages? This would be great for debugging.

**Reply by Navid200** [reply]
Oh, I forgot.
You should not test this with your real setup forcing yourself to have low forecasts.  
  
xDrip has a fake data source.  You should leave your main active xDrip alone and use a test phone for this.  And this explains how you can use fake data source:  
https://navid200.github.io/xDrip/docs/FakeDataSource.html  

As far as what makes sense here, I was thinking of adding one new element to the xDrip legend.  The new element will determine the color of the note when the forecast low alert triggers, currently red.  
You added the ability to change the color of blood glucose reading text on the main screen.  
This will be the color of the forecast low text.  
  
As I said already, the forecast low text appears in three different colors.  But, when the forecast low alert triggers, there is only one color.  It's that one that we need to add an option for the user for.  
  
You have given the user the ability to choose the text color and I assume it was because different phones and different individuals may have a hard time seeing a particular color combination.  

Red on black provides a very low contrast even for someone who has no color blindness issues at all.  
So, it makes sense, at least to me, to offer the option also for the forecast low alert considering the text size is even smaller.
